bellows/zigbee/endpoint.py

```diff
@@ -59,22 +61,29 @@ def initialize(self):
 
         self.output_clusters = sd.output_clusters
 
+        for cluster in sd.output_clusters:
+            self.add_cluster(cluster)
+
```

Comment:
I have a branch which provides both input and output clusters in separate attributes. Combining them like this doesn't seem like the right thing to do.